src/routes/transactions/entities/human-description.entity.ts

```diff
@@ -0,0 +1,25 @@
+import {
+  InfoFragment,
+  ValueType,
+} from '@/domain/human-description/entities/human-description.entity';
+
+export interface RichInfoFragment extends InfoFragment {
+  richData: Record<string, unknown> | null;
+}
+
+export interface RichTokenValueFragment extends RichInfoFragment {
+  type: ValueType.TokenValue;
+  value: string;
+  richData: {
+    symbol: string | null;
+    logoUri: string | null;
+  };
+}
+
+export interface RichTextFragment extends RichInfoFragment {}
+export interface RichAddressFragment extends RichInfoFragment {}
+export interface RichNumberFragment extends RichInfoFragment {}
+
+export type RichInfo = {
+  fragments: RichInfoFragment[];
+};
```

Comment:
It will be useful if we decide to add another property like the one proposed in the API spec for multisend support. In that case we would only have to change it in one place.